device-types/Cisco/C9200L-48PL-4X.yaml

```diff
@@ -0,0 +1,133 @@
+comments: '[Cisco Catalyst 9200 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-9200-series-switches/nb-06-cat9200-ser-data-sheet-cte-en.html)'
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: usb
+    type: usb-mini-b
+interfaces:
+  - name: GigabitEthernet1/0/1
+    type: 1000base-t
+  - name: GigabitEthernet1/0/2
+    type: 1000base-t
+  - name: GigabitEthernet1/0/3
+    type: 1000base-t
+  - name: GigabitEthernet1/0/4
+    type: 1000base-t
+  - name: GigabitEthernet1/0/5
+    type: 1000base-t
+  - name: GigabitEthernet1/0/6
+    type: 1000base-t
+  - name: GigabitEthernet1/0/7
+    type: 1000base-t
+  - name: GigabitEthernet1/0/8
+    type: 1000base-t
+  - name: GigabitEthernet1/0/9
+    type: 1000base-t
+  - name: GigabitEthernet1/0/10
+    type: 1000base-t
+  - name: GigabitEthernet1/0/11
+    type: 1000base-t
+  - name: GigabitEthernet1/0/12
+    type: 1000base-t
+  - name: GigabitEthernet1/0/13
+    type: 1000base-t
+  - name: GigabitEthernet1/0/14
+    type: 1000base-t
+  - name: GigabitEthernet1/0/15
+    type: 1000base-t
+  - name: GigabitEthernet1/0/16
+    type: 1000base-t
+  - name: GigabitEthernet1/0/17
+    type: 1000base-t
+  - name: GigabitEthernet1/0/18
+    type: 1000base-t
+  - name: GigabitEthernet1/0/19
+    type: 1000base-t
+  - name: GigabitEthernet1/0/20
+    type: 1000base-t
+  - name: GigabitEthernet1/0/21
+    type: 1000base-t
+  - name: GigabitEthernet1/0/22
+    type: 1000base-t
+  - name: GigabitEthernet1/0/23
+    type: 1000base-t
+  - name: GigabitEthernet1/0/24
+    type: 1000base-t
+  - name: GigabitEthernet1/0/25
+    type: 1000base-t
+  - name: GigabitEthernet1/0/26
+    type: 1000base-t
+  - name: GigabitEthernet1/0/27
+    type: 1000base-t
+  - name: GigabitEthernet1/0/28
+    type: 1000base-t
+  - name: GigabitEthernet1/0/29
+    type: 1000base-t
+  - name: GigabitEthernet1/0/30
+    type: 1000base-t
+  - name: GigabitEthernet1/0/31
+    type: 1000base-t
+  - name: GigabitEthernet1/0/32
+    type: 1000base-t
+  - name: GigabitEthernet1/0/33
+    type: 1000base-t
+  - name: GigabitEthernet1/0/34
+    type: 1000base-t
+  - name: GigabitEthernet1/0/35
+    type: 1000base-t
+  - name: GigabitEthernet1/0/36
+    type: 1000base-t
+  - name: GigabitEthernet1/0/37
+    type: 1000base-t
+  - name: GigabitEthernet1/0/38
+    type: 1000base-t
+  - name: GigabitEthernet1/0/39
+    type: 1000base-t
+  - name: GigabitEthernet1/0/40
+    type: 1000base-t
+  - name: GigabitEthernet1/0/41
+    type: 1000base-t
+  - name: GigabitEthernet1/0/42
+    type: 1000base-t
+  - name: GigabitEthernet1/0/43
+    type: 1000base-t
+  - name: GigabitEthernet1/0/44
+    type: 1000base-t
+  - name: GigabitEthernet1/0/45
+    type: 1000base-t
+  - name: GigabitEthernet1/0/46
+    type: 1000base-t
+  - name: GigabitEthernet1/0/47
+    type: 1000base-t
+  - name: GigabitEthernet1/0/48
+    type: 1000base-t
+  - name: GigabitEthernet1/1/1
+    type: 1000base-t
```

Comment:
Looking at the spec sheet these are SFP+ ports so should be `type: 10gbase-x-sfpp`